network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/AbstractBranchImpl.java

```diff
@@ -135,6 +133,72 @@ public CurrentLimitsAdder newCurrentLimits2() {
         return new CurrentLimitsAdderImpl<>(Branch.Side.TWO, this);
     }
 
+    @Override
+    public ApparentPowerLimitsAdder newApparentPowerLimits1() {
+        return new ApparentPowerLimitsAdderImpl<>(Branch.Side.ONE, this);
+    }
+
+    @Override
+    public ApparentPowerLimitsAdder newApparentPowerLimits2() {
+        return new ApparentPowerLimitsAdderImpl<>(Side.TWO, this);
+    }
+
+    @Override
+    public ApparentPowerLimits getApparentPowerLimits1() {
+        return resource.getAttributes().getApparentPowerLimits1() != null ? new ApparentPowerLimitsImpl(this, resource.getAttributes().getApparentPowerLimits1()) : null;
```

Comment:
Is it ok to create a new object every time ? In some other code, we store the object as a field to avoid that